actor/src/lib.rs

```diff
@@ -289,10 +291,14 @@ pub trait Actor: Send + Sized + 'static {
     }
 
     /// At start hook of actor
-    async fn on_start(&mut self, _ctx: &mut Context<Self>) {}
+    async fn on_start(&mut self, ctx: &mut Context<Self>) {
+        let _ = ctx;
+    }
 
     /// At stop hook of actor
-    async fn on_stop(&mut self, _ctx: &mut Context<Self>) {}
+    async fn on_stop(&mut self, ctx: &mut Context<Self>) {
+        let _ = ctx;
```

Comment:
what's the point of this?

Reply:
In order to make signature look nice. If you generate HTML with `cargo doc` there will be `ctx` in argument name, not `_ctx`.